src/index.js

```diff
@@ -0,0 +1,132 @@
+const state = {
+  temp: 68
+};
+
+let isFahrenheit = true;
+
+const vibe = {
+  sheet: 2
+}
+
+let isReset = false;
+
+const changeTemp = (direction) => {
+  if (direction === 'up') {
+    state.temp += 1;
+  } else if (direction === 'down') {
+    state.temp -= 1;
+  } else if (direction === null) {
+    pass
+  }
+
+  const temperature = document.querySelector("#temperature");
+  if (isFahrenheit) {
+    temperature.textContent = `${state.temp}℉`;
+  } else {
+    temperature.textContent = `${(state.temp -32) * 5/9}℃`;
+  }
+  changeTempFontColor();
+}
+
+
+const swapStyleSheet = (event) => {
+
+  if (vibe.sheet === 1 || vibe.sheet === 2) {
+      vibe.sheet += 1;
+  } else if (vibe.sheet === 3) {
+      vibe.sheet = 1;
+  }
+
+  document.getElementById("vibe").setAttribute("href", `styles/vibe${vibe.sheet}.css`)
+}
+
+const changeTempFontColor = () => {
+  if (state.temp <= 49) {
+    color = "#3F50B5";
+    landscape = "🤧⛄️🌲⛄️🛷🌲⛷🗻🌲⛸🏂⛄️🍂❄️";
+  } else if (state.temp >= 50 && state.temp<= 59) {
```

Comment:
Notice that we don't actually need to check the lower bound of the range. If we reached this check, then the prior check for whether the temp was below 49 must have failed, meaning it's at least 50.